### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by 29td**
### Project Milestone : Metrics Webapp

**_Day 1_**

- Select the API you are going to use.
- Build files structure for your React app.
- Prepare routes and navigation in your app.

**_Day 2_**

- Make sure that a user can display a list of items and filter them. Data should come from the API.
- Create the tests for the application.

**_Day 3_**

- Style your components to match the design provided.
- Deploy the project and test for final details.
- Record a video for your project.
- Create a good README and PR description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
-# matrics-webapp
-This capstone project is about building a mobile web application to check a list of metrics (numeric values) that I will create making use of React and Redux.
+# Project Name
```
Comment (DammyShittu) [inline README.md:1]: - [x] Kindly include the link to your video in the readme documentation

- [x] Kindly deploy your app and include the link in the readme also

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @29td,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/States.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  return (
+    <ul className="cards">
+      {states.map((state) => (
+        <li className="cards__item" key={state}>
+          <Link className="cards__link" to={`/${state}`}>
+            {state}
+          </Link>
+        </li>
+      ))}
+    </ul>
```
Comment (DammyShittu) [inline src/pages/States.js]: - [x] Your home component should include a filtering functionality for searching items on the home page.

  gim

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:✔️✔️

## Hi @29td 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️
- app works well✔️
- Validations correctly implemented✔️
- No linter errors✔️
- Livelink ✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**